Gulpfile.js

```diff
@@ -734,14 +734,23 @@ export function functional(done) {
 }
 
 export function e2eRun(done) {
-  const result = spawnSync(
-    "npx",
-    ["playwright", "test", "-c", "tests/e2e/playwright.config.ts", "tests/e2e"],
-    {
-      stdio: "inherit",
-      shell: IS_WINDOWS,
-    },
-  );
+  // Get all arguments after 'gulp e2e'
```

Comment:
```suggestion
  // Get all arguments after 'gulp e2e -t'
```
Is this more accurate? It isn't checking for any other flags, just whatever comes after `-t`, right?

Reply:
That comment is stale, I was originally passing down all arguments as is.

Reply:
fixed the comment